silence/silence.go

```diff
@@ -106,6 +108,50 @@ type Silences struct {
 	st  gossipData
 }
 
+type metrics struct {
+	gcDuration       prometheus.Summary
+	snapshotDuration prometheus.Summary
+	queriesTotal     prometheus.Counter
+	queryErrorsTotal prometheus.Counter
+	queryDuration    prometheus.Histogram
+}
+
+func newMetrics(r prometheus.Registerer) *metrics {
+	m := &metrics{}
+
+	m.gcDuration = prometheus.NewSummary(prometheus.SummaryOpts{
+		Name: "alertmanager_silences_gc_duration_seconds",
+		Help: "Duration of the last garbage collection cycle.",
```

Comment:
"last silence garbage collection cycle" for people to not confuse the HELP string with general Go garbage collection?